src/views/PurchaseOrder.vue

```diff
@@ -17,55 +17,70 @@
 
         <ion-list>
           <ion-list-header>{{ $t("Select the column index for the following information in the uploaded CSV.") }}</ion-list-header>
+
+          <ion-item lines="none">
+            <ion-label>{{ $t("Select mapping") }}</ion-label>
+            <ion-select interface="popover" @ionChange="mapFields">
+              <ion-select-option v-for="mapping in fieldMappings" :value="mapping" :key="mapping">{{ mapping.name }}</ion-select-option>
```

Comment:
Use a unique value as a key instead of using an object.

Reply:
Made the requested change